native/cocos/bindings/jswrapper/v8/Utils.cpp

```diff
@@ -155,29 +156,44 @@ static void warnWithinTimesInReleaseMode(const char *msg) {
 
 template <typename T>
 void setReturnValueTemplate(const Value &data, const T &argv) {
-    if (data.getType() == Value::Type::Undefined) {
-        argv.GetReturnValue().Set(v8::Undefined(argv.GetIsolate()));
-    } else if (data.getType() == Value::Type::Null) {
-        argv.GetReturnValue().Set(v8::Null(argv.GetIsolate()));
-    } else if (data.getType() == Value::Type::Number) {
-        argv.GetReturnValue().Set(v8::Number::New(argv.GetIsolate(), data.toDouble()));
-    } else if (data.getType() == Value::Type::BigInt) {
-        constexpr int64_t maxSafeInt = 9007199254740991LL;  // value refer to JS Number.MAX_SAFE_INTEGER
-        constexpr int64_t minSafeInt = -9007199254740991LL; // value refer to JS Number.MIN_SAFE_INTEGER
-        if (data.toInt64() > maxSafeInt || data.toInt64() < minSafeInt) {
-            // NOTICE: Precision loss will happend here.
-            warnWithinTimesInReleaseMode<100>("int64 value is out of range for double");
-            CC_ABORT(); // should be fixed in debug mode.
+    switch (data.getType()) {
+        case Value::Type::Undefined: {
+            argv.GetReturnValue().Set(v8::Undefined(argv.GetIsolate()));
+            break;
+        }
+        case Value::Type::Null: {
+            argv.GetReturnValue().Set(v8::Null(argv.GetIsolate()));
+            break;
+        }
+        case Value::Type::Number: {
+            argv.GetReturnValue().Set(v8::Number::New(argv.GetIsolate(), data.toDouble()));
+            break;
+        }
+        case Value::Type::BigInt: {
+            constexpr int64_t maxSafeInt = 9007199254740991LL;  // value refer to JS Number.MAX_SAFE_INTEGER
+            constexpr int64_t minSafeInt = -9007199254740991LL; // value refer to JS Number.MIN_SAFE_INTEGER
+            if (data.toInt64() > maxSafeInt || data.toInt64() < minSafeInt) {
+                // NOTICE: Precision loss will happend here.
+                warnWithinTimesInReleaseMode<100>("int64 value is out of range for double");
+                CC_ABORT(); // should be fixed in debug mode.
+            }
+            argv.GetReturnValue().Set(v8::Number::New(argv.GetIsolate(), static_cast<double>(data.toInt64())));
+            break;
+        }
+        case Value::Type::String: {
+            v8::MaybeLocal<v8::String> value = v8::String::NewFromUtf8(argv.GetIsolate(), data.toString().c_str(), v8::NewStringType::kInternalized);
```

Comment:
Changed from `kNormal` to `kInternalized` which depends `hash` to find whether there is a same string in the cache.

Reply:
Probably, I needed to revert this change since we don't want all of `returning value` being cached.
Could refer to https://groups.google.com/g/v8-users/c/xoqi4ee8x74?pli=1

Reply:
Have reverted to kNormal.